tests/test_cli.py

```diff
@@ -3,13 +3,14 @@
 import re
 import shutil
 from contextlib import contextmanager
+from functools import partialmethod
```

Comment:
This is not supported in Python 2.7, where it would be `from functools import partial`

Reply:
I added some boilerplate code to handle the import in Python 2.7.